### ISSUE TITLE
Cannot copy image from explorer to chat

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#239370]
From @jooyoungseo:

Screen reader users often navigate files in the explorer.

1. Use `ctrlCmd+c` to copy an image from the explorer
2. Focus the chat input
3. Use `ctrlCmd+v` to paste the image
4. 🐛 instead of attaching the image, the file name is pasted

**Comment by justschen** [comment]
to confirm, this is with the OS native explorer correct? 

in VS Code, if you hover over the tab of the image and paste it should work fine, but we should probably handle the OS scenario. it can be helpful if we're able to add not only images, but also diff file types by copying and pasting perhaps?

**Comment by meganrogge** [comment]
This is using only the keyboard - move focus to the explorer, copy using the keyboard, focus the chat panel input, paste.
